init-tools.sh

```diff
@@ -110,11 +118,25 @@ if [ ! -e $__PROJECT_JSON_FILE ]; then
     echo $__PROJECT_JSON_CONTENTS > "$__PROJECT_JSON_FILE"
 
     if [ ! -e $__BUILD_TOOLS_PATH ]; then
+        echo "Running: $__DOTNET_CMD restore "$__PROJECT_JSON_FILE" --packages $__PACKAGES_DIR --source $__BUILDTOOLS_SOURCE" >> $__init_tools_log
         $__DOTNET_CMD restore "$__PROJECT_JSON_FILE" --packages $__PACKAGES_DIR --source $__BUILDTOOLS_SOURCE
     fi
+    
+    if [ ! -e "$__BUILD_TOOLS_PATH/init-tools.sh" ]; then
+        echo "ERROR: Could not restore build tools correctly." 1>&2
+        display_error_message
+    fi
 
+    echo "Initializing BuildTools..."
+    echo "Running: $__BUILD_TOOLS_PATH/init-tools.sh $__scriptpath $__DOTNET_CMD $__TOOLRUNTIME_DIR" >> $__init_tools_log
     # On ubuntu 14.04, /bin/sh (symbolic link) calls /bin/dash by default.
     $__BUILD_TOOLS_PATH/init-tools.sh $__scriptpath $__DOTNET_CMD $__TOOLRUNTIME_DIR
```

Comment:
Similar to my earlier comment, consider redirecting output to the log. Refer to https://github.com/dotnet/corefx/blob/master/init-tools.sh#L113